shells/pdksh/Portfile

```diff
@@ -14,13 +16,14 @@ long_description	\
 	initially created by Eric Gisin using Charles Forsyth's public \
 	domain V7 shell as a base as well as parts of the BRL shell. \
 	Since then a number of people have maintained and contributed to it
+
 platforms	darwin
-homepage	http://www.cs.mun.ca/~michael/pdksh/
-master_sites	ftp://ftp.cs.mun.ca/pub/pdksh/ \
-		ftp://gd.tuwien.ac.at/utils/shells/pdk \
-		ftp://ftp.lip6.fr/pub/unix/shells/pdksh/ \
-		ftp://ftp.rge.com/pub/shells/pdksh/
-checksums	md5 871106b3bd937e1afba9f2ef7c43aef3
+homepage    ftp://ftp.lip6.fr/pub/unix/shells/pdksh/
+master_sites	ftp://gd.tuwien.ac.at/utils/shells/pdk \
+		ftp://ftp.lip6.fr/pub/unix/shells/pdksh/
+checksums           rmd160  3ecb47ac7c7fc43445806a68c740c94278e2368f \
+                    sha256  ab15bcdd50f291edc632bca351b2edce5405d4f2ce3854d3d548d721ab9bbfa6
```

Comment:
Since you are updating the checksums, also add the new size parameter.

Reply:
port -v checksum doesn't produce this. Does it do so on mater?

Reply:
It does now (https://github.com/macports/macports-base/commit/95a153e6a4bff8b99502806960c916a2b5d20504).